3.train_ebimage/1.random_forest/random-forest.py

```diff
@@ -0,0 +1,41 @@
+# %% Initialization
```

Comment:
Major question: How is this file executed with 6 different model initializations. Since we adding bounds to the F1 score estimates (which is great!) we should add instructions on how this code was executed 6 times. It is unclear to me how you've done it in this case.